elements/line-status.js

```diff
@@ -3,12 +3,11 @@ const html = Tram.html({
   'line-status-line': require('../elements/line-status-line')
 })
 
-module.exports = (attrs) => {  
+module.exports = (attrs) => {
   return html`
-    <div class="box" style="width: 100%; min-height: 300px; margin: 20px;${attrs.style || ''}">
+    <div class="box" style="width: 100%; min-height: 300px; margin: 20px 10px;${attrs.style || ''}">
```

Comment:
changed margin to be `20px 10px` (making vertical margin 20px, and horizontal margin 10px; this evens out since you have 10px from the right side of one block, and the left side of the next block)